.circleci/docker/common/install_conda.sh

```diff
@@ -71,6 +71,15 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
     conda_install magma-cuda100 -c pytorch
   elif [[ "$CUDA_VERSION" == 10.1* ]]; then
     conda_install magma-cuda101 -c pytorch
+  elif [[ "$CUDA_VERSION" == 10.2* ]]; then
```

Comment:
I'm about to land this change as part of https://github.com/pytorch/pytorch/pull/38747

Reply:
Thanks for the heads up. How do I get the new tag for the `DOCKER_IMAGE_TAG`?

Reply:
It's a hash sum of CircleCI setup job on your PR. Please make sure to add new image to https://github.com/pytorch/pytorch/blob/master/.circleci/verbatim-sources/workflows/workflows-ecr-gc.yml#L13

And to trigger build on PR, delete `triggers:` subsection from https://github.com/pytorch/pytorch/blob/master/.circleci/verbatim-sources/workflows/workflows-docker-builder.yml#L2

Reply:
> It's a hash sum of CircleCI setup job on your PR. Please make sure to add ...

How do I find/calculate the hash sum?

Reply:
@malfet: how do I find/calculate the hash sum?

Reply:
@malfet I removed the `triggers:` section and now see a few more `ecr_gc_job_for*` CI jobs, but none of them seem to be docker builds. What am I missing?

Reply:
There are hints in gh-38686, it seems the checksum is `DOCKER_TAG=$(git rev-parse HEAD:.circleci/docker)`. But perhaps this PR should wait for that one which will make all this easier.

Reply:
it seems not so simple. `git rev-parse HEAD:.circleci/docker` gives a hash as a string, for instance `0b8f43b056e9b40f7b7b0747883d200e36fcd227`, but the needed tag is of the format `209062ef-ab58-422a-b295-36c4eed6e906`